### PR TITLE
Improve documentation filenames and titles

### LINKED ISSUE #162: Canviar enllaços base a castellà
Les rutes HTML dels fitxers .MD són en català però potser seria millor tenir-ho en castellà per tal de que l'usuari que llegeixi la documentació en castellà es pugui guiar una "mica" millor.

### PR DESCRIPTION
**Pull request by gdalmau**
Closes #162

This is to change names and directories to spanish

### CODE REVIEW

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_improve_documentation_filenames_and_titles/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_improve_documentation_filenames_and_titles/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_improve_documentation_filenames_and_titles/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_improve_documentation_filenames_and_titles/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_improve_documentation_filenames_and_titles/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_improve_documentation_filenames_and_titles/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_improve_documentation_filenames_and_titles/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_improve_documentation_filenames_and_titles/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_improve_documentation_filenames_and_titles/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_improve_documentation_filenames_and_titles/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_improve_documentation_filenames_and_titles/

File: README.md [hunk 1]
```diff
@@ -99,12 +99,15 @@ Markdown extensions:
 
 ## Project structure
 
+!!! note
```
Comment (jaumef) [inline README.md]: Github no te admonition, pel que no hi ha notes

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_improve_documentation_filenames_and_titles/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_improve_documentation_filenames_and_titles/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_improve_documentation_filenames_and_titles/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_improve_documentation_filenames_and_titles/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_improve_documentation_filenames_and_titles/

**Pull request merged**